src/components/sync-connected-sites.tsx

```diff
@@ -71,6 +72,15 @@ export function SyncConnectedSites( {
 		return siteSections;
 	}, [ connectedSites ] );
 
+	const showPullConfirmation = useConfirmationDialog( {
+		localStorageKey: 'dontShowPullConfirmation',
+		message: __( 'Overwrite Studio site' ),
```

Comment:
I see that on some other modals we are using the site name instead of `Studio site` - what do you think about passing the actual site name in the title? (I see it is not in the design so happy to proceed without this change but might be nice to have this for consistency).

Reply:
I think it's important to keep "Studio" to refer it's the local site what we will overwrite. If we use the site name we have high chances that the production site name will be the same

Happy to change it if you are still thinking it's clearer to use the site name.

Reply:
@katinthehatsite, @wojtekn , can you give it a second thought ?

Reply:
This is a good point, thanks @sejas . Let's leave it as is, then. 

Later, we can review all dialogs holistically and unify them.